packages/amplify-appsync-simulator/src/server/operations.ts

```diff
@@ -27,9 +27,22 @@ export class OperationServer {
     this._app.use(cors());
     this._app.post('/graphql', this.handleRequest);
     this._app.get('/api-config', this.handleAPIInfoRequest);
+    this._app.delete('/clear-data', this.handleClearDBData);
     this._app.use('/', express.static(STATIC_ROOT));
   }
 
+  private handleClearDBData = async (request: express.Request, response: express.Response) => {
+    console.log('Clearing DB data...');
+    try {
+      const deletedItems = await this.simulatorContext.clearData();
+      console.log('DB data cleared');
+      return response.status(200).send({ message: `Successfully deleted ${JSON.stringify(deletedItems)} tables` });
+    } catch (e) {
+      console.log('Error clearing DB data');
+      return response.status(500).send({ message: e.message });
```

Comment:
I'd include the error message in your console log here, and should use `console.error` instead.